hvac/v1/__init__.py

```diff
@@ -385,6 +385,69 @@ def auth_userpass(self, username, password, mount_point='userpass', use_token=Tr
 
         return self.auth('/v1/auth/{}/login/{}'.format(mount_point, username), json=params, use_token=use_token)
 
+    def create_userpass(self, username, password, policies, mount_point='userpass'):
+        """
+        POST /auth/<mount point>/users/<username>
+        """
+
+        # Users can have more than 1 policy. It is easier for the user to pass in the
+        # policies as a list so if they do, we need to convert to a , delimited string.
+        if type(policies) == list:
```

Comment:
Instead of `if type(policies) == list:`, consider `if isinstance(policies, (list, set, tuple)):` to capture `set` and `tuple` types too? Or the following (nicer, imo) syntax:

~~~
def create_userpass(self, username, password, mount_point='userpass', *policies):
    .. snip ..
    policies = ','.join(policies)
~~~

and do away with type checking altogether. Same would go for the `create_app_id` and `create_user_id` below.